src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/3.sql

```diff
@@ -1,33 +1,24 @@
--- NOTE: This script DROPS AND RECREATES all database objects.
 -- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
-
 /*************************************************************
-    Configure database
+Wrapping up in the multiple transactions except CREATE FULLTEXT INDEX which is non-transactional script.
```

Comment:
nit: Since we are  no longer adding multiple transaction, could we remove this comment.